src/control/control.h

```diff
@@ -36,6 +36,23 @@ class ControlDoublePrivate : public QObject {
     // TODO: don't expose this implementation detail
     constexpr static double kDefaultValue = 0.0;
 
+    struct Parameters {
+        ConfigKey key;
+        ControlObject* pCreatorCO = nullptr;
+        QString name{};
+        QString description{};
```

Comment:
{} is redundant. 
```suggestion
        QString name;
        QString description;
```

Reply:
Technically yes, though adding it always simplifies the language IMO since the reader would not have to know whether a typename is a primitive or a class type. The {} initializes both.

Reply:
Following this rule we will clutter the whole Mixxx code with many of these {}. Since now we always remove redundant initializes in code reviews. Let's stick to it for now.

Reply:
I'm not saying we should apply to all of mixxx, but I think it would make sense for aggregate classes. Its also only redundant in the sense that its explicit, it doesn't result in extra instructions (unlike redundant initialization in a ctor body). I would really prefer to keep it at least for aggregates because it clearly says "this member can't ever be uninitialized".

Reply:
I really don't mind in this case. I am just afraid future discussions in other cases. IMHO this does not improve readability, because of the following example:  Let's imagine is an unknown type. Without initialization the value might be uninitialized or default initialized. To see the difference you need to look up the implementation that is out of scope here. So you need to do a lookup in both cases. 

In https://github.com/mixxxdj/mixxx/pull/13586 (Still waiting for merge b.t.w.) we had: 
`mixxx::audio::FramePos startPosition`
vs:
`mixxx::audio::FramePos startPosition = mixxx::audio::kInvalidFramePos;`
the later avoids the lookup. 

using here 
`mixxx::audio::FramePos startPosition{};` 
will no longer inform the reader that the position is initialized as invalid.

Reply:
> Without initialization the value might be uninitialized or default initialized.

Right, using `type member{}` explicitly alleviates that.

> In https://github.com/mixxxdj/mixxx/pull/13586 (Still waiting for merge b.t.w.) we had:

Yes, I actually prefer the latter, I just didn't want to argue on that PR. 

> using here
mixxx::audio::FramePos startPosition{};
will no longer inform the reader that the position is initialized as invalid.


But neither does `mixxx::audio::FramePos startPosition;`...

I think these are two different cases. In the case in this PR I just say "I want this default initialized" explicitly. In the `FramePos` example you brought up, we rather want to say "I want this framepos to be initialized with a known constant value". In the framepos example, incidentally the default ctor sets it to the same value sure, so you can write it in both ways, but one of them is more expressive than the other. 

Either way, we should not introduce rules to always do X (such as "we don't initialize default constructed classes") because that removes the ability for the code author to appropriately express their intent when writing the code.